docs/modules/ROOT/pages/tutorial/relayer.adoc

```diff
@@ -0,0 +1,154 @@
+# Using Relayer for Sending Transactions
+
+In this tutorial, we will explore how to use the xref::manage.adoc#relayers[Relayer] to send transactions. We will cover:
+
+* Checking relayer information.
+* Sending a transaction.
+* Checking the relayer status.
+
+By the end of this tutorial, you will have a basic understanding of how to use the Relayer to interact with smart contracts.
+
+[[pre-requisites]]
+== Pre-requisites
+
+* OpenZeppelin Defender account. You can sign up to Defender https://defender.openzeppelin.com/v2/?utm_campaign=Defender_2.0_2023&utm_source=Docs#/auth/sign-up[here, window=_blank].
+* https://nodejs.org/en[NodeJS and NPM, window=_blank]
+* https://www.npmjs.com/package/ts-node[Typescript for Node js, window=_blank]
+* Any IDE or text editor
+
+[[Set-Up-Relayer]]
+== 1. Set Up Relayer
+
+Let's start by creating a Relayer:
+
+* Open https://defender.openzeppelin.com/v2/#/relayers[Defender Operate & Automate] in a web browser.
++
+image::tutorial-relayer-step1.png[Manage Relayer]
+* Click on *Create Relayer* with the name ETH Sepolia Relayer and Sepolia network.
++
+image::tutorial-relayer-step2.png[Create Relayer]
+* Fund it with some Sepolia ETH. This relayer will send and pay for the automated transactions.
+* To create an API key for a Relayer, click on the Relayer and then on the *More* button to expand the dropdown and select *Create API Key*
++
+image::tutorial-relayer-step3.png[Create Relayer API]
+* Now you can set API key expiration in minutes, hours or days.
++
+image::tutorial-relayer-step3-1.png[Save Relayer API]
+* Once the API Key is created, make sure to write down the secret key. The API secret is only visible once during the creation — if you don’t write it down, it’s lost forever.
++
+image::tutorial-relayer-step4.png[Save Relayer API]
+
+[[Check-Relayer-Information]]
+== 2. Check Relayer Information
+
+*  Let's start by checking the information of our relayer.
+*  Add Relayer in `.env` File
+*  Edit `.env` file in your project root directory and add your Relayer API Key and Secret:
++
+[source,jsx]
+----
+RELAYER_API_KEY=your_api_key
+RELAYER_SECRET_KEY=your_api_secret
+----
+
+* Create a file named `storeObject.ts` in the project. 
+* Now let’s add the following code:
++
+[source,jsx]
+----
+const { Defender } = require('@openzeppelin/defender-sdk');
+
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+async function main() {
+
+const client = new Defender({
+
+relayerApiKey: process.env.RELAYER_API_KEY,
+relayerApiSecret: process.env.RELAYER_SECRET_KEY,
+});
+
+const info = await client.relaySigner.getRelayer();
+console.log('Relayer Info:', JSON.stringify(info, null, 2));
+
+}
+
+main().catch((error) => {
+console.error(error);
+process.exitCode = 1;
+});
+
+----
+
+* Execute the script to check the relayer information:
++
+[source,jsx]
+----
+ts-node storeObject.ts
```

Comment:
I would add `node storeObject.js` as well if folks wants to use node directly instead of ts-node